cOS/cOS.py

```diff
@@ -317,21 +317,23 @@ def getFrameRange(path):
 def normalizeFramePadding(filepath):
 	fileInfo = getPathInfo(filepath)
 
-	if len(fileInfo['basename'].split('.')) >= 3:
-		framePadding = fileInfo['basename'].split('.')[-2]
-		hashReg = re.compile('#+')
-		dollarReg = re.compile('\$F[1-9]')
-		if hashReg.match(framePadding):
-			padding = framePadding.count('#')
+	if not len(fileInfo['basename'].split('.')) >= 3:
+		return filepath
```

Comment:
shouldn't be dependent on name.padding.exr

should work with name_padding.exr as well